src/gridjs.svelte

```diff
@@ -89,6 +89,27 @@
       instance.render(node);
     }
   });
+
+  $: if (node) {
+    instance
+      .updateConfig({
+        from,
+        data,
+        columns,
+        server,
+        search,
+        sort,
+        pagination,
+        language,
+        width,
+        height,
+        autoWidth,
+        fixedHeader,
+        style,
+        className,
```

Comment:
missing `resizable`, I think to make it into variables would be better

Reply:
Resizable got flagged in the TS types as not existent, but i can add it back in.